### PR TITLE
Reliance time fix

### PR DESCRIPTION
**Pull request by jayaraopratik**

### CODE REVIEW

File: ner_v1/detectors/temporal/time/time_detection.py [hunk 1]
```diff
@@ -1467,3 +1471,29 @@ def _remove_time_range_entities(self, time_list, original_list):
...
+                'time_type': None
+            }
+
+            original_list.append(original)
+            time_list.append(time)
+
+        return time_list, original_list
+
+
+
```
Comment (srajdev) [inline ner_v1/detectors/temporal/time/time_detection.py]: W391 blank line at end of file

File: ner_v2/detectors/temporal/time/standard_time_regex.py [hunk 2]
```diff
@@ -239,6 +240,34 @@ def _detect_hour_minute(self, time_list, original_list):
...
+        for pattern in patterns:
+            t1 = pattern[1]
+            t2 = pattern[2]
+            original = pattern[0]
+
+            if len(t1) <= 2 and len(t2) <= 2:
+
+                time = {
+                    'hh': int(t1),
+                    'mm': int(t2),
```
Comment (krupalmodi18) [inline ner_v2/detectors/temporal/time/standard_time_regex.py]: nn not considered

File: ner_v2/detectors/temporal/time/standard_time_regex.py [hunk 3]
```diff
@@ -239,6 +240,34 @@ def _detect_hour_minute(self, time_list, original_list):
 
         return time_list, original_list
 
+    def _detect_time_with_coln_format(self, time_list, original_list):
```
Comment (krupalmodi18) [inline ner_v2/detectors/temporal/time/standard_time_regex.py]: add docs

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Pull request merged**